src/Stratis.Bitcoin.IntegrationTests/Connectivity/ConnectivityTests.cs

```diff
@@ -23,8 +26,38 @@ public ConnectivityTests()
         }
 
         [Fact]
-        public async Task WhenConnectingWithAddnodeConnectToPeerAndAnyPeersInTheAddressManager()
+        public void WhenStartingNodeWithAListOfIPAddressesTheAddressesArePropagatedToAnotherPeer()
         {
+            // TS100_Connectivity_CanGetIPList.
+
+            using (NodeBuilder builder = NodeBuilder.Create(this))
+            {
+                CoreNode node1 = builder.CreateStratisPosNode(this.Network).NotInIBD().Start();
+                CoreNode node2 = builder.CreateStratisPosNode(this.Network).NotInIBD().Start();
+
+                var node1DNSSeeds = node1.FullNode.Network.DNSSeeds;
+                var node2DNSSeeds = node2.FullNode.Network.DNSSeeds;
+
+                node1DNSSeeds.Should().BeEmpty();
+                node2DNSSeeds.Should().BeEmpty();
+
+                string hostname = string.Empty;
+
+                for (int i = 0; i < 10; i++)
+                {
+                    hostname = $"testnet{i}.stratisplatform.com";
+                    node2DNSSeeds.Add(new DNSSeedData(hostname, hostname));
+                }
+
+                node1DNSSeeds.Should().HaveSameCount(node2DNSSeeds);
```

Comment:
The name of this test seems to imply it is doing something that is not reflected in the test itself, unless I am mistaken.

Reply:
Ok.  Also addeding the correct implementation for this after @zeptin's comments.

_`getaddr` messages were sent from node 2 to node 1, and that node 1 sent a payload back to node 2 with peers in it (the `addr` payload)._